selfdrive/car/chrysler/carcontroller.py

```diff
@@ -23,25 +23,13 @@ def __init__(self, dbc_name, CP, VM):
   def update(self, CC, CS):
     can_sends = []
 
-    # TODO: can we make this more sane? why is it different for all the cars?
-    lkas_control_bit = self.lkas_control_bit_prev
-    if CS.out.vEgo > self.CP.minSteerSpeed:
-      lkas_control_bit = True
-    elif self.CP.carFingerprint in (CAR.PACIFICA_2019_HYBRID, CAR.PACIFICA_2020, CAR.JEEP_CHEROKEE_2019):
-      if CS.out.vEgo < (self.CP.minSteerSpeed - 3.0):
-        lkas_control_bit = False
-    elif self.CP.carFingerprint in RAM_CARS:
-      if CS.out.vEgo < (self.CP.minSteerSpeed - 0.5):
-        lkas_control_bit = False
-
     # EPS faults if LKAS re-enables too quickly
-    lkas_control_bit = lkas_control_bit and (self.frame - self.last_lkas_falling_edge > 200)
-    lkas_active = CC.latActive and self.lkas_control_bit_prev
+    lkas_active = CC.latActive and (self.frame - self.last_lkas_falling_edge > 200)
```

Comment:
@adeebshihadeh safe to get rid of this if latActive considers lower min steer speed? The difference here is it won't send the bit when disabled any more. If you don't know I can try to find a tester